sample_apps/simple_app_under_oe/host/host.cc

```diff
@@ -140,15 +144,15 @@ int main(int argc, const char* argv[])
 
     if (ret) {
         printf("%s succeeded!\n", argv[2]);
+        rv = 0;
     } else {
         printf("%s failed!\n", argv[2]);
     }
 
-
 exit:
     // Clean up the enclave if we created one
     if (enclave)
         oe_terminate_enclave(enclave);
 
-    return ret;
+    return 0;
```

Comment:
Change `ret` to always return 0; otherwise, even due to benign failures this program will return non-zero $rc, causing shell script to exit early.

Reply:
Done. `return rv` also seems to work reliably now in all test scenarios.